gatorgrouper/utils/read_student_file.py

```diff
@@ -13,17 +13,30 @@ def read_student_file(filepath):
 
     # read the raw CSV data
     with open(filepath, "r") as csvfile:
-        csvdata = list(csv.reader(csvfile, delimiter=","))
+        has_header = csv.Sniffer().has_header(csvfile.read(1024))
 
+    with open(filepath, "r") as csvfile:
```

Comment:
Could you change the argument to be called `csvfile` and check `type(csvfile)` at the start of the function to see if a path string or a file object is being passed in instead of always opening the file from a path string? In the `feature/mvp` branch we ran into a situation where we wanted to call this function but only had access to an object and not a path, and it would probably be better to change it here rather than duplicate code over there.

Reply:
Hi @aubreypc, I don't think this should be handled in this PR.